### PR TITLE
Pull Request for #16: Make heatmaps compatible with SK

### PR DESCRIPTION
**Pull request by rcalvo12**
In this issue, I edited the heatmap function in `_diagnostics.py` to allow users to make heatmaps using the SK distance as well as the TV distance. 

I also added a bounds and vrange argument so that users can zoom into the plot with more control and control the color scaling of the heatmap. These are very easy to remove if necessary.

### CODE REVIEW

File: src/BootstrapReport/_diagnostics.py [hunk 1]
```diff
@@ -7,29 +7,40 @@
 
 class DiagnosticsMixin:
     
-    def _heat_map(self, density = 50, outfile = False):
+    def _heat_map(self, distance = "TV", density = 50, bounds = [], vrange = [0,1], outfile = False):
```
Comment (jmshapir) [inline src/BootstrapReport/_diagnostics.py:10]:
```suggestion
    def _heat_map(self, distance = "SK", density = 50, bounds = [], vrange = [0,1], outfile = False):
```

File: src/BootstrapReport/_diagnostics.py [hunk 2]
```diff
@@ -7,29 +7,40 @@
...
+            table = [[helpers.get_sk_dist(replicates, norm(loc = mu, scale = sigma)) for sigma in sigma_range] for mu in mu_range]
+        
+        df = pd.DataFrame(table, columns = sigma_label, index = mu_label)
+                
+        ax = sns.heatmap(df, vmin = vrange[0], vmax = vrange[1])
         ax.invert_yaxis()
         ax.set_xlabel('σ')
         ax.set_ylabel('µ')
-        plt.title('TVD with varied µ and σ')
+        plt.title(f'{distance}D with varied µ and σ')
```
Comment (jmshapir) [inline src/BootstrapReport/_diagnostics.py:43]:
```suggestion
        plt.title(f'{distance} with varied µ and σ')
```

**Review by jmshapir — CHANGES_REQUESTED** [review]
@rcalvo12 thanks!

Had a small suggestion on the plot title and also suggesting to switch default behavior to plotting SK.

Adding @MosesStewart for code review.

Thanks both!

**Review by MosesStewart — APPROVED** [review]

**Comment by MosesStewart** [comment]
@rcalvo12 all the code changes look good to me!

**Comment by jmshapir** [comment]
@MosesStewart thanks!

If we accept my suggested edits does it look like the code will run successfully?

If so can you implement those edits and wrap up the pull please?

Thanks!

**Comment by jmshapir** [comment]
@MosesStewart fyi you can prioritize https://github.com/JMSLab/BootstrapReport/pull/17#issuecomment-1817278185 ahead of #18 thanks.

**Comment by MosesStewart** [comment]
> @MosesStewart thanks!
> 
> If we accept my suggested edits does it look like the code will run successfully?
> 
> If so can you implement those edits and wrap up the pull please?
> 
> Thanks!

Sorry ~ I just noticed this comment. The changes look correct ~ indistinguishable from a `python` master!

**Review by jmshapir — APPROVED** [review]

**Comment by MosesStewart** [comment]
Summary in https://github.com/JMSLab/BootstrapReport/issues/16#issuecomment-1818011649

**Pull request merged**